netutils/thttpd/cgi-src/ssi.c

```diff
@@ -99,81 +99,82 @@ static void internal_error(char *reason)
 {
   char *title = "500 Internal Error";
 
-  printf("\
-<HTML><HEAD><TITLE>%s</TITLE></HEAD>\n\
-<BODY><H2>%s</H2>\n\
-Something unusual went wrong during a server-side-includes request:\n\
-<BLOCKQUOTE>\n\
-%s\n\
-</BLOCKQUOTE>\n\
-</BODY></HTML>\n", title, title, reason);
+  printf("<HTML><HEAD><TITLE>%s</TITLE></HEAD>\n"
+    "<BODY><H2>%s</H2>\n"
+    "Something unusual went wrong during a server-side-includes request:\n"
+    "<BLOCKQUOTE>\n"
+    "%s\n"
+    "</BLOCKQUOTE>\n"
+    "</BODY></HTML>\n",
+    title, title, reason);
```

Comment:
```suggestion
  printf("<HTML><HEAD><TITLE>%s</TITLE></HEAD>\n"
         "<BODY><H2>%s</H2>\n"
         "Something unusual went wrong during a server-side-includes request:\n"
         "<BLOCKQUOTE>\n"
         "%s\n"
         "</BLOCKQUOTE>\n"
         "</BODY></HTML>\n",
         title, title, reason);
```

Reply:
can't since the line 104 will exceed 79 chars.